e2e/parent-arg-from-func-call/src/Preg.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class Preg
+{
+    /**
+     * @param string $subject
```

Comment:
The thing that make Scope not exists even only 1 rule run is this `@param string` and then there scalar and array check...